plone/app/contenttypes/browser/tabular.py

```diff
@@ -0,0 +1,80 @@
+from Products.Five import BrowserView
+
+# BEWARE: the cell views are registered for ContentListingObject
+# which are not acquisition aware.
+# That precludes using Products.Five.ViewPageTemplateFile
+# and imposes to use zope.browserpage.viewpagetemplatefile.
+from zope.browserpage.viewpagetemplatefile import ViewPageTemplateFile
+# BEWARE
+
+class TitleCell(BrowserView):
+    __call__ = ViewPageTemplateFile("templates/titlecell.pt")
+
+    @property
+    def title(self):
+        return self.context.Title() or self.context.getId()
+
+    @property
+    def link(self):
+        suffix = (
+            "/view"
+            if self.context.PortalType in self.table_view.use_view_action
+            else ""
+        )
+        return self.context.getURL() + suffix
+
+    @property
+    def type_class(self):
+        return ("contenttype/" +
+                self.table_view.normalizeString(self.context.PortalType())
```

Comment:
You call `PortalType` here, but a few lines above, you use it as attribute instead of method. One is probably wrong.

I never use PortalType. I think I have only seen this sometimes in the portal_catalog.
In both cases I would use `portal_type`.